neuralmagic/benchmarks/configs/benchmark_serving.json

```diff
@@ -0,0 +1,39 @@
+{
+	"configs": [
+		{
+			"description": "Benchmark vllm serving",
+			"dataset_download_cmds": [
+				"wget https://huggingface.co/datasets/anon8231489123/ShareGPT_Vicuna_unfiltered/resolve/main/ShareGPT_V3_unfiltered_cleaned_split.json"
+			],
+			"models": [
+				"facebook/opt-125m",
+				"PY007/TinyLlama-1.1B-step-50K-105b",
```

Comment:
Lets remove anything bigger than 7b until we have the structure setup and infra running. We can add the bigger models after.

Also use the following for Tinyllama:
- `TinyLlama/TinyLlama-1.1B-Chat-v1.0`

Reply:
Agreed. Dont have to worry about the script stopping due to GPU OOM for the time being.